sdk/ml/azure-ai-ml/azure/ai/ml/_utils/utils.py

```diff
@@ -1418,3 +1418,14 @@ def extract_name_and_version(azureml_id: str) -> Dict[str, str]:
         "name": name,
         "version": version,
     }
+
+
+def _get_evaluator_properties():
+    return {
```

Comment:
Just curious why we need two flags. Is "is-evaluator" not enough for filtering?

Reply:
We need other one to filter then from model catalog. Other all evalautors will show up in model catalog